ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -216,6 +255,102 @@ KEXT_STATIC void UseMainForkIfNamedStream(
     }
 }
 
+static void ResizePendingRenames(uint32_t newMaxPendingRenames)
+{
+    PendingRenameOperation* newArray = Memory_AllocArray<PendingRenameOperation>(newMaxPendingRenames);
+    PendingRenameOperation* arrayToFree = nullptr;
+    uint32_t arrayToFreeLength = 0;
+    
+    SpinLock_Acquire(s_renameLock);
+    {
+        if (newMaxPendingRenames >= s_maxPendingRenames)
```

Comment:
Should this just be '>'?

Reply:
Yes, it probably should. We should probably also not be reinventing array resizing so many times… (As I've mentioned in other PRs, my preferred option would be a resizeable array template.)

Reply:
This should be fixed now.